build-system/tasks/e2e/describes-e2e.js

```diff
@@ -174,7 +187,7 @@ function getFirefoxArgs(config) {
   const args = [];
 
   if (config.headless) {
-    args.push('-headless');
+    args.push('--headless');
```

Comment:
Nice catch!

Does this mean we no longer need `xvfb` In`.travis.yml`?

Reply:
Oh maybe! Let me see.

Reply:
Yup no need for it as long as we use headless. @prateekbh fyi!